src/fxa-geodb.js

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// these are defaults, can be overloaded by configuring options
+var path = require('path');
+
+var DEFAULTS = require(path.join('..', 'lib', 'defaults'));
+var ERRORS = require(path.join('..', 'lib', 'errors'));
+var maxmind = require('maxmind');
+var Promise = require('bluebird');
+
+module.exports = function (options) {
+  'use strict';
+
+  options = options || {};
+  var dbPath = options.dbPath || DEFAULTS.DEFAULT_DB_PATH;
+
+  var dbLookup, locationData;
```

Comment:
do you need to define `locationData` here or can it be done below when you fetch it